SchoolAPI/Controllers/StudentController.cs

```diff
@@ -39,8 +39,6 @@ public Student Get(int id)
         [HttpPost]
         public IActionResult Post([FromBody] Student student)
         {
-            student.StudentID = _context.Students.Select(student => student.StudentID).Max() + 1;
```

Comment:
Removed the line as it isn't really required. ID will be generated automatically